### ISSUE TITLE
How to call base methods from Frappe (removeDataPoint...)

### ISSUE THREAD

**Issue by ruiisidro** [himynameisdave/svelte-frappe-charts#19]
I have a reference like this:
```
let myChart;
 ... 

<Chart bind:this={myChart} {...chartConfigs} />
```
but when calling 

```
 myChart.removeDataPoint();

```
I get a myChart.removeDataPoint is not a function

**Comment by himynameisdave** [comment]
Great question, currently this functionality has not been added but it really should be! This would not be terribly different from when we implemented `export` functionality here: https://github.com/himynameisdave/svelte-frappe-charts/pull/11/files

I'll open a PR to include these [other base API methods](https://frappe.io/charts/docs/reference/api) soon (unless you want to).

**Comment by himynameisdave** [comment]
@ruiisidro updated, [published in `1.3.0`](https://github.com/himynameisdave/svelte-frappe-charts/releases/tag/v1.3.0) 🎉 

Let me know how it works for you / if you have any other suggestions for how to improve this library. Thanks!

**Comment by ruiisidro** [comment]
@himynameisdave Thanks! 👍
